src/transformState.js

```diff
@@ -4,8 +4,26 @@
  * @param {Object} state
  * @param {Object[]} actions
  */
+
 function transformState(state, actions) {
   // write code here
+  actions.forEach(action => {
+    switch (action.type) {
+      case 'addProperties':
+        Object.assign(state, action.extraData);
+        break;
+      case 'removeProperties':
+        action.keysToRemove.forEach(key => delete state[key]);
+        break;
+      case 'clear':
+        Object.keys(state).forEach(key => delete state[key]);
+        break;
+      default:
+        break;
```

Comment:
I would add here some return string, for catching errors, in case you get different type.🤔
```suggestion
      default:
      	return 'Undefined action type: ' action.type; 
        break;
```

Reply:
@Olexandr6 
Great comment and error handling  👍